### PR TITLE
Simulation fixes for fleet (mostly VTOL)

### PR DESCRIPTION
**Pull request by LorenzMeier**
@AndreasAntener @tumbili @sanderux @dagar The models in SITL seemed mostly unusable. I've fixed a number of issues with them and the flight performance improved significantly (from unusable to reasonably realistic). Cross-testing, in particular under joystick control, would be most appreciated.

And Merry Christmas! I guess this is my gift for the PX4 VTOL developers.

Note that the control gains and stall speeds became more realistic with these fixes. That's a good hint towards things going into the right direction.

### CODE REVIEW

**Comment by dagar** [comment]
Thanks! I'll give them a try.
As a result I was using SITL for initial development, then graduate to X-Plane HIL for testing (when possible).

File: posix-configs/SITL/init/ekf2/standard_vtol [hunk 1]
```diff
@@ -35,12 +35,16 @@ param set MPC_XY_FF 0.1
 param set MPC_Z_VEL_MAX 1.5
 param set MPC_Z_VEL_P 0.6
 param set MPC_Z_VEL_I 0.15
+param set FW_AIRSPD_MIN 14
+param set FW_AIRSPD_TRIM 20
```
Comment (sanderux) [inline posix-configs/SITL/init/ekf2/standard_vtol]: I would propose 16m/s here, 4% pitch is what SITL does and that is quite accurate with reality.

**Comment by jgoppert** [comment]
@Lorenz thanks!

**Comment by LorenzMeier** [comment]
I'm continuing this here because I cleaned up the commits: https://github.com/PX4/Firmware/pull/6168

**Pull request closed without merging**